### ISSUE TITLE
Install instructions are not enough clear

### ISSUE THREAD

**Issue by ooii** [gugoan/economizzer#46]
Hi,

I'm trying to install economizzer but I'm facing the following (noobie?) issues:
- Importing the economizzer.sql file. How should one achieve that? I created a database named economizzer like you require. Then, I try to import the file by running `mysqlimport -u root -pmypassword --local economizzer.sql` and I have the following error `mysqlimport: Error: 1146, Table 'economizzer.economizzer' doesn't exist, when using table: economizzer`
- Also, the last line of your Install instructions is `Access **http://economizzer/web** with user and password...`. But, how to start it? Where one could configure the port etc? 
Anyway, thanks a lot for that great and nice software.

**Comment by ooii** [comment]
*For the first point, I created the user in the db, I granted him access of the database and then entered all the content of economizzer.sql in mysql shell. It seems OK.
*For the second point, I think I have to put it in the web folder, usually `/var/www/`, and then access `http://myip/economizzer/web`.

I'm now able to reach economizzer but it does not work. I have the following error:
```
Unknown Property – yii\base\UnknownPropertyException

Getting unknown property: amnah\yii2\user\models\forms\LoginForm::username
```
The whole log details are available at http://pastebin.com/JyT9AcRP
Note that there is an `$_SESSION` where appears my other software sessions (like Plex) so I did not attach it. If needed, I can send them separately.
And I tried that on OSX, synology, ubuntu, with either Apache or nginx.
